app/code/core/Mage/GoogleAnalytics/Block/Ga.php

```diff
@@ -475,18 +475,18 @@ protected function _getOrdersTrackingCodeAnalytics4()
                     $_item = [
                         'item_id' => $item->getSku(),
                         'item_name' => $item->getName(),
-                        'quantity' => $item->getQtyOrdered(),
-                        'price' => $item->getBasePrice(),
-                        'discount' => $item->getBaseDiscountAmount()
+                        'quantity' => (int) $item->getQtyOrdered(),
```

Comment:
does quantity has to be int? what about decimal quantities which openmage supports?

Reply:
It seems that only support integer. With 2.00 it was reading it as 1 by default as if it wasn't valid.
And every change needs to be tested after 24 hours, so it's hard for me

Reply:
ok let's leave it this way, google's documentation is unclear: https://developers.google.com/analytics/devguides/collection/ga4/reference/events since it says "number", which doesn't mean much, it's used with decimals somewhere but the quantity seems to be an integer... google...